superset/assets/spec/javascripts/explore/components/ControlPanelsContainer_spec.jsx

```diff
@@ -45,12 +46,12 @@ describe('ControlPanelsContainer', () => {
 
   it('renders ControlPanelSections', () => {
     wrapper = shallow(<ControlPanelsContainer {...defaultProps} />);
-    expect(wrapper.find(ControlPanelSection)).toHaveLength(6);
+    expect(wrapper.find(ControlPanelSection)).toHaveLength(2);
```

Comment:
is this going to break people's charts if the behavior of the tests changed? It's not clear to me why a refactor would require the tests to be altered.

Reply:
Nope, this is because the old code expected control panel for `table` to be registered. 
`table` has 4 sections of control + default 2 sections = 6
when using `SCOPED_FILTER`, it adds another section so it became 7. 

With the update in this PR to delegate the registration of control panels to during setup, the unit tests did not register control panel for table, so it misses the 4 sections.
Anyway, I updated the unit test to register the control panel config for table, so the unit tests will keep the name numbers.